src/components/policy/test/include/mock_pt_ext_representation.h

```diff
@@ -112,7 +115,7 @@ class MockPTExtRepresentation : public MockPTRepresentation,
                      void(const std::string& app_id,
                           const std::string& type,
                           const std::string& value));
-  MOCK_CONST_METHOD3(Add,
+  MOCK_CONST_METHOD3(}  // namespace policy_managerAdd,
```

Comment:
Just confused is there something mistake in this line, I mean ```( }```, seems missed mock_object ```Add```?